website/src/routes/api/VariableSizeGrid.js

```diff
@@ -96,39 +96,59 @@ const PROPS = [
 const METHODS = [
   {
     description: (
-      <p>
-        <code>VariableSizeGrid</code> caches offsets and measurements for each
-        column index for performance purposes. This method clears that cached
-        data for all columns after (and including) the specified index. It
-        should be called whenever a column's width changes. (Note that this is
-        not a typical occurrance.)
-      </p>
+      <Fragment>
+        <p>
+          <code>VariableSizeGrid</code> caches offsets and measurements for each
+          column index for performance purposes. This method clears that cached
+          data for all columns after (and including) the specified index. It
+          should be called whenever a column's width changes. (Note that this is
+          not a typical occurrance.)
+        </p>
+        <p>
+          You can set <code>shouldForceUpdate</code> to <code>false</code>
+          to prevent the list calling <code>forceUpdate</code> internally.
```

Comment:
I don't think this is enough of an explanation. Maybe something more like:

> By default the grid will automatically re-render after the index is reset. If you would like to delay this re-render until e.g. a state update has completed in the parent component, specify a value of `false` for the second, optional parameter.

(And similar for the below blocks.)